### DISCUSSION TITLE
How does this compare to WayDroid?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [vinegarhq/sober#448] [Q&A] [answered]
How does this compare to merely installing [`play.google.com/store/apps/details?id=com.roblox.client`](https://play.google.com/store/apps/details?id=com.roblox.client&hl=en_GB) via [`waydro.id/#install`](https://waydro.id/#install)? [^1]

[^1]: https://www.reddit.com/r/linux_gaming/comments/1enbhrs/comment/lyl0xlh/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by tunis4** [comment, marked as answer]
- Waydroid runs an entire Android system and then the Roblox app inside of it, while Sober does bare minimum translation of the Android APIs that Roblox needs into Linux APIs, so Sober is running basically like a native Linux port of Roblox.
- Sober makes Roblox behave like its running on desktop instead of mobile, meaning you will get PC controls and UIs, and you will be able to join PC-only games. It also has tweaks such as making mouse locking work properly.
- Sober allows Roblox to use the native Vulkan API directly, unlike Waydroid which (as far as I know) runs an entire graphics driver that is incompatible with Nvidia and has worse performance.
- Sober is way easier to setup.
